bands/templates/band_admin/index.html

```diff
@@ -105,7 +114,11 @@
             success: function (response) {
                 if (response.id) {
                     $band.empty().append(
-                        '<button class="btn btn-sm btn-primary btn-mark" data-group_id="' + groupId + '" data-group_name="' + groupName + '">Mark as Band</button>'
+                        '<select class="form-control" class="group_type" name="group_type">'+
+                        '<option value="">Select a group type</option>'+
+                        `{{ options(c.GROUP_TYPE_OPTS) }}`+
```

Comment:
Did you know that the latest version of JS accepts a ` tag to accompany multiple string lines? I didn't! We need that for this macro because it generates a newline.

Reply:
Oooh, how widely supported is this in browsers? We may want to avoid bleeding edge JS features. Looks like it's totally broken in IE 11 😞

Reply:
nooooooooooooo IE whyyyy

In that case I'm not totally sure how to include an `{{ options }}` tag inside JS like this. The output returned by the macro is on two (or more) lines...

Reply:
Ah, what I have done in the past (I don't know if this is actually a good idea 😅) is render the `{{ options }}` inside a hidden div with `display: none`. Then in the javascript I can grab the rendered `<options>` using innerHTML